packages/website/src/pages/index.tsx

```diff
@@ -78,53 +68,53 @@ const features: FeatureItem[] = [
-            on turning your source code into a data format called an AST in
-            order to do their jobs.
-          </p>
-        </div>
-        <div className="col col--offset-2 col--8">
-          <p>
-            However, it turns out that ESLint and TypeScript use different ASTs
-            to each other.
+            <strong>
+              <code>typescript-eslint</code> enables ESLint to run on TypeScript
+              code.
+            </strong>{' '}
+            It brings in the best of both tools to help you write the best
+            JavaScript or TypeScript code you possibly can.
           </p>
         </div>
         <div className="col col--offset-2 col--8">
           <p>
-            The reason for this difference is not so interesting or important
-            and is simply the result of different evolutions, priorities, and
-            timelines of the projects.
+            ESLint and TypeScript represent code differently internally.
+            ESLint's default JavaScript parser cannot natively read in
+            TypeScript-specific syntax and its rules don't natively have access
+            to TypeScript's type information.
           </p>
         </div>
         <div className="col col--offset-2 col--8">
           <p>
-            This project, <code>typescript-eslint</code>, exists primarily
-            because of this major difference between the projects.
-          </p>
-        </div>
-        <div className="col col--offset-2 col--8">
-          <p>
-            <code>typescript-eslint</code> exists so that you can use ESLint and
-            TypeScript together, without needing to worry about implementation
-            detail differences wherever possible.
+            <code>typescript-eslint</code>:
+            <ul>
+              <li>allows ESLint to parse TypeScript syntax</li>
+              <li>
+                creates a set of tools for ESLint rules to be able to use
+                TypeScript's type information
+              </li>
+              <li>
+                provides a large list of lint rules that are specific to
+                TypeScript and/or use that type information
+              </li>
+            </ul>
           </p>
         </div>
       </div>
     ),
   },
 ];
 
-function Feature({ imageUrl, title, description }: FeatureItem): JSX.Element {
-  const imgUrl = imageUrl ? useBaseUrl(imageUrl) : undefined;
+function Feature({ title, description }: FeatureItem): JSX.Element {
   return (
     <div className="col col--12 padding-vert--lg">
-      {imgUrl && (
-        <div className="text--center">
-          <img className={styles.featureImage} src={imgUrl} alt={title} />
-        </div>
-      )}
       <h2 className="text--center">{title}</h2>
       {description}
+      <div className={styles.buttons}>
+        <Link className="button button--primary" to={useBaseUrl('docs/')}>
+          Get Started
+        </Link>
+      </div>
```

Comment:
Learning from working in product land: you want a CTA (Call To Action) visible on the screen at all times. Even if it's as simple as repeating the same basic button below each section, it helps make sure users both always know what to do and are visually encouraged to do it.

Reply:
bro, do you even get started?

![image](https://user-images.githubusercontent.com/7462525/141262550-8cdc520d-fe44-4f79-9c00-e94cb8b05ed7.png)